_layouts/post.html

```
@@ -2,87 +2,25 @@
+  {% if page.subtitle %}
+  <p class="post-subtitle">{{ page.subtitle }}</p>
+  {% endif %}
+  
+  <p class="post-meta">
+    {{ page.date | date: "%B %d, %Y" }}
+    {% if page.author %} • {{ page.author }}{% endif %}
+  </p>
+  
+  {{ content }}
+  
+  {% if page.tags.size > 0 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)